components/brave_rewards/browser/publisher_info_database.h

```diff
@@ -39,6 +39,14 @@ class PublisherInfoDatabase {
             int limit,
             const ledger::PublisherInfoFilter& filter,
             ledger::PublisherInfoList* list);
+  int Count(const ledger::PublisherInfoFilter& filter);
+
+  std::string BuildClauses(int start,
```

Comment:
sorry, one more small change. I think these should be private?

Reply:
Makes sense to me